camel-kafka-connector-catalog/src/generated/resources/connectors/camel-ahc-sink.json

```diff
@@ -0,0 +1,162 @@
+{
+	"connector": {
+		"class": "org.apache.camel.kafkaconnector.ahc.CamelAhcSinkConnector",
+		"artifactId": "camel-ahc-kafka-connector",
+		"groupId": "org.apache.camel.kafkaconnector",
+		"id": "camel-ahc-sink",
+		"type": "sink",
+		"version": "0.6.0-SNAPSHOT"
+	},
+	"properties": {
+		"camel.sink.path.httpUri": {
+			"name": "camel.sink.path.httpUri",
+			"description": "The URI to use such as http:\/\/hostname:port\/path",
+			"defaultValue": "null",
+			"priority": "HIGH"
+		},
+		"camel.sink.endpoint.bridgeEndpoint": {
+			"name": "camel.sink.endpoint.bridgeEndpoint",
+			"description": "If the option is true, then the Exchange.HTTP_URI header is ignored, and use the endpoint's URI for request. You may also set the throwExceptionOnFailure to be false to let the AhcProducer send all the fault response back.",
+			"defaultValue": "false",
+			"priority": "MEDIUM"
+		},
+		"camel.sink.endpoint.bufferSize": {
+			"name": "camel.sink.endpoint.bufferSize",
+			"description": "The initial in-memory buffer size used when transferring data between Camel and AHC Client.",
+			"defaultValue": "4096",
+			"priority": "MEDIUM"
+		},
+		"camel.sink.endpoint.connectionClose": {
+			"name": "camel.sink.endpoint.connectionClose",
+			"description": "Define if the Connection Close header has to be added to HTTP Request. This parameter is false by default",
+			"defaultValue": "false",
+			"priority": "MEDIUM"
+		},
+		"camel.sink.endpoint.cookieHandler": {
+			"name": "camel.sink.endpoint.cookieHandler",
+			"description": "Configure a cookie handler to maintain a HTTP session",
+			"defaultValue": "null",
+			"priority": "MEDIUM"
+		},
+		"camel.sink.endpoint.headerFilterStrategy": {
+			"name": "camel.sink.endpoint.headerFilterStrategy",
+			"description": "To use a custom HeaderFilterStrategy to filter header to and from Camel message.",
+			"defaultValue": "null",
```

Comment:
Isn't a problem to make the difference between the `null` as `String` and no default value?

I'm guessing here that it means there is no default value defined

Reply:
Yes, there is no default value when it's null. This is something coming from the how kafka connect define the options.

Reply:
it means that we cannot define a default value which is the null String? or there are ways to escape it?

Reply:
I think we can improve the connector aurogeneration for this. @valdar cc

Reply:
@oscerd it seems that we can use `ConfigDef.NO_DEFAULT_VALUE` kafka connect side to distinguish between no default value and `null` as default value. I am investigating if that information is available in the vanilla camel catalog that we process to generate connectors.